.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```

Comment:
These lines and the `.gitignore` file are generated by Husky when [creating a hook](https://typicode.github.io/husky/#/?id=create-a-hook).